packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -140,14 +129,11 @@ export const Collapse: FunctionComponent<Props> = ({
   const panelClass = cx([style.collapse, 'panel-container', className]);
   const loaderClass = loading ? cx([style.loader, style.loaderActive]) : cx([style.loader]);
   const headerClass = collapsible ? cx([style.header]) : cx([style.headerCollapsed]);
-  const headerButtonsClass = collapsible ? cx([style.headerButtons]) : cx([style.headerButtonsCollapsed]);
 
   return (
     <div className={panelClass}>
       <div className={headerClass} onClick={onClickToggle}>
-        <div className={headerButtonsClass}>
-          <Icon name={isOpen ? 'angle-up' : 'angle-down'} />
-        </div>
+        {collapsible && <Icon name={isOpen ? 'angle-up' : 'angle-down'} />}
```

Comment:
I removed the wrapper and it's classes. Seems like the main point was just hiding the icon and when not hidden the positioning css created this:
![Screenshot from 2021-05-17 22-32-10](https://user-images.githubusercontent.com/1014802/118644996-a8599d00-b7de-11eb-9707-a1f9af5d495c.png)